### PR TITLE
Acceptance test for creating password with emoji

### PR DESCRIPTION
**Pull request by bhawanaprasain**
## Description
Users can create their password including emojis.

Part of issue #34437 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

### CODE REVIEW

**Comment by bhawanaprasain** [comment]
@dpakach @paurakhsharma  @skshetry  please review on my PR .

File: tests/acceptance/features/apiProvisioning-v1/resetUserPassword.feature [hunk 1]
```diff
@@ -71,3 +71,13 @@ Feature: reset user password
     And the HTTP status code should be "200"
     And the content of file "textfile0.txt" for user "brand-new-user" using password "%alt1%" should be "ownCloud test text file 0" plus end-of-line
     But user "brand-new-user" using password "%regular%" should not be able to download file "textfile0.txt"
+
+  Scenario: reset user password including emoji
```
Comment (paurakhsharma) [inline tests/acceptance/features/apiProvisioning-v1/resetUserPassword.feature]: Maybe it's better to create Scenario Outline for this?

File: tests/acceptance/features/apiProvisioning-v2/resetUserPassword.feature [hunk 2]
```diff
@@ -71,3 +71,13 @@ Feature: reset user password
     And the HTTP status code should be "200"
     And the content of file "textfile0.txt" for user "brand-new-user" using password "%alt1%" should be "ownCloud test text file 0" plus end-of-line
     But user "brand-new-user" using password "%regular%" should not be able to download file "textfile0.txt"
+
+  Scenario: reset user password including emoji
```
Comment (paurakhsharma) [inline tests/acceptance/features/apiProvisioning-v2/resetUserPassword.feature]: Maybe it's better to create Scenario Outline for this?

**Review by skshetry — COMMENTED** [review]
:+1:

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@bhawanaprasain please backport to `stable10`

**Comment by phil-davis** [comment]
Backport `stable10` #35431

**Pull request merged**